src/components v2/Atoms/Forms/RadioButtonGroup/RadioButtonGroup.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <p-input :errors="errors">
+    <p-label>
+      {{ label }}
+      <slot v-if="!label" name="label" />
+    </p-label>
+    <div class="radio-wrapper" :style="numRows">
+      <P2RadioButton
+        v-for="(item, index) in items"
+        :id="`${item.label}-${id}`"
+        :key="`${item.label}-${id}`"
+        :checked="index === selectedIndex"
+        :label="item.label"
+        :rows="numRows"
+        :disabled="disabled"
+        :icon="item.icon"
+        @click="() => $emit('click', index)"
+      ></P2RadioButton>
```

Comment:
**nitpick:** use shorthand close syntax.

you can close the component by writing `/>`.